components/cdc/src/initializer.rs

```diff
@@ -606,11 +606,15 @@ impl<E: KvEngine> Initializer<E> {
 
         let hint_min_ts = self.checkpoint_ts.into_inner();
         let (mut total_count, mut filtered_count, mut tables) = (0, 0, 0);
-        collection.iter_user_collected_properties(|prop| {
+        collection.iter_table_properties(|table_prop| {
```

Comment:
I just flattened the original helper function. Since I updated the table property iteration signature.